src/views/saphyr/SaphyrReception.vue

```diff
@@ -48,20 +48,31 @@ export default {
     async handleSampleExtractionTubes() {
       try {
         let getSETubeResponse = await this.getSampleExtractionTubesForBarcodes(this.getBarcodes())
-        if (!getSETubeResponse.successful || getSETubeResponse.empty) {
-          throw getSETubeResponse.errors
+
+        if (!getSETubeResponse.success || getSETubeResponse.data.data.length === 0) {
+          let errorMessage = 'Sample Extraction tubes failed to be imported'
+          // Is this just for supporting the tests??
+          if (getSETubeResponse.errors && getSETubeResponse.errors.length > 0) {
+            errorMessage = getSETubeResponse.errors
+          }
+          throw { message: errorMessage }
         }
 
         let exportSampleTubesResponse = await this.exportSampleExtractionTubesIntoTraction(
           this.sampleExtractionTubes,
         )
 
-        if (!exportSampleTubesResponse.successful || exportSampleTubesResponse.empty) {
-          throw exportSampleTubesResponse.errors
+        if (
```

Comment:
I think we could refactor this further based on what we learnt doing the pooling but something for another day ...